src/Utilities/Utilities.jl

```diff
@@ -363,12 +363,31 @@ end
 
 # Finding URLs -- based partially on code from the main Julia repo in `base/methodshow.jl`.
 
```

Comment:
I'm likely to just copy-paste whatever you suggest, so easier to get the git attribution right if you edit as you please.